client/webserver/site/src/js/settings.js

```diff
@@ -61,6 +63,27 @@ export default class SettingsPage extends BasePage {
     })
   }
 
+  async showDexInfo (page) {
```

Comment:
I've left some comments on the implementation of this function, but really, I don't think you should be using javascript for this at all. Do it all in the template.

Reply:
Everything is being done in template now.